Model/Sync/Category/Processor/ProcessByCategory.php

```diff
@@ -140,30 +140,24 @@ public function processEntity($retryCategoryIds = [])
         $currentTime = date('Y-m-d H:i:s');
         $batchSize = $this->config->getConfig('product_sync_limit');
         $existColls = [];
-        $attributeId = $this->data->getAttributeId(Config::CATEGORY_SYNC_ATTR_CODE);
-        $collection = $this->categoryCollectionFactory->create();
-        $collection->addAttributeToSelect('*');
-        $collection->addNameToResult();
+        $collection = $this->categoryHelper->getStoreCategories(true, true);
         if ($retryCategoryIds) {
-            $collection->getSelect()
-                ->where('e.entity_id in (?)', $retryCategoryIds);
+            /** @phpstan-ignore-next-line */
```

Comment:
@manaftryzens Can you explain the additions of `/** @phpstan-ignore-next-line */`?